test/plot-schema.json

```diff
@@ -44148,6 +44148,17 @@
      },
      "description": "The links of the Sankey plot.",
      "editType": "calc",
+     "hovercolor": {
+      "arrayOk": true,
+      "description": "Sets the `link` hover color. It can be a single value, or an array for specifying hovor colors for each `link`. If `link.hovercolor` is omitted, then by default, links will become solid grey when hovered over.",
```

Comment:
```suggestion
      "description": "Sets the `link` hover color. It can be a single value, or an array for specifying hover colors for each `link`. If `link.hovercolor` is omitted, then by default, links will become solid grey when hovered over.",
```